### PR TITLE
:bookmark: (24.9.0)

### PR DESCRIPTION
**Pull request by MatissJanis**
- web: https://github.com/actualbudget/actual/pull/3348
- server: https://github.com/actualbudget/actual-server/pull/444
- docs: https://github.com/actualbudget/docs/pull/444

### CODE REVIEW

File: blog/2024-09-03-release-24.9.0.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+authors: MatissJanis
+---
+
+The release has the following notable improvements:
+
+- mobile: long-press transactions to reveal more actions
+- transactions table: show imported payee on hover of payee column
+- filtering: ability to filter by "has tags" conditions
+- budgets: highlight current month
+- imports: ability to ship heading lines
```
Comment (matt-fidd) [inline blog/2024-09-03-release-24.9.0.md:17]:
```suggestion
- imports: ability to skip heading lines
```

File: docs/releases.md [hunk 2]
```diff
@@ -1,5 +1,140 @@
...
 
+## 24.9.0
+
+The release has the following notable improvements:
+
+- mobile: long-press transactions to reveal more actions
+- transactions table: show imported payee on hover of payee column
+- filtering: ability to filter by "has tags" conditions
+- budgets: highlight current month
+- imports: ability to ship heading lines
```
Comment (matt-fidd) [inline docs/releases.md:11]:
```suggestion
- imports: ability to skip heading lines
```

**Review by matt-fidd — CHANGES_REQUESTED** [review]
One typo :)

File: blog/2024-09-03-release-24.9.0.md (hunk 1 above)
Comment (MatissJanis) [reply blog/2024-09-03-release-24.9.0.md:17]: 🚢

**Review by matt-fidd — APPROVED** [review]
Looking good!

**Review by twk3 — APPROVED** [review]

**Review by MikesGlitch — APPROVED** [review]

**Pull request merged**